product_abc_classification/views/product_view.xml

```diff
@@ -60,21 +60,52 @@
             </field>
         </field>
     </record>
-    <record id="product_template_form_view" model="ir.ui.view">
+    <record id="product_template_only_form_view" model="ir.ui.view">
```

Comment:
Why do you change this view? It has been a topic of discussion at other PR https://github.com/OCA/product-attribute/pull/623#pullrequestreview-630027873 to make the classification at the product level or template level. But for the moment it is done at the product level, so I think we should maintain the original view at least for now.

Reply:
You can still doing the classification at product level https://github.com/OCA/product-attribute/pull/1013/files#diff-04d0b42ea9b4d2ea95cdccfeb237f9ed5e4fcfd956e7b524ec91c3e4143e5535R89 

I do this change because with the previous implementation when the `product.template` had more than one variants, we could not change the value of the product level from his variants. Now, if the product has more than one variants, we are able to change the value from his variants and not from the template.

Reply:
Ok! Thanks for the explanation!